doc/tutorials/using_podman_instead_of_docker.md

```diff
@@ -0,0 +1,100 @@
+# Introduction
+
+Podman is an alternative container engine which does not require root
+privileges on the host OS.
+
+Podman aims at being compatible with Docker and in most cases simply replacing
+`docker` by `podman` is enough.
+
+However, there are some notable differences when running containers themselves
+as a user other than root.
+
+## Who Is This Guide For?
+
+Anyone who uses Podman to run their containers.
+
+## Prerequisites
+
+- A working Podman installation
+
+## Basics
+
+In most cases one can simply replace `docker` by `podman` when running a command.
+
+For instance, to build the Debian bullseye container with Docker the command is
+
+```sh
+docker build \
+  --tag "build-elektra-debian-bullseye" \
+	--build-arg "JENKINS_USERID=$(id -u)" \
+	--build-arg "JENKINS_GROUPID=$(id -g)" \
+	--file "scripts/docker/debian/bullseye/Dockerfile" \
+	scripts/docker/debian/bullseye/
+```
+
+Simply replacing `docker` with `podman` works in this case
+
+```sh
+podman build \
+  --tag "build-elektra-debian-bullseye" \
+	--build-arg "JENKINS_USERID=$(id -u)" \
+	--build-arg "JENKINS_GROUPID=$(id -g)" \
+	--file "scripts/docker/debian/bullseye/Dockerfile" \
+	scripts/docker/debian/bullseye/
+```
+
+## Running a container as a user other than root
+
+One area where Podman notably differs from Docker is when interacting with the
+filesystem of the host.
+
+This is relevant when mounting directories of the host filesystem as volumes.
+
+With Docker the container engine itself runs as a priviledged process and
+therefore the permissions in the host OS are not relevant.
+
+With Podman the situation is more involved, as the container engine does not
+run with root priviledges.
+
+For instance, the containers specified for building Elektra are configured such
+that the user inside the container is a non-root user.
+
+If one wishes to mount the Elektra source directory from the host filesystem as
+a volume for the container, extra steps are necessary.
+
+Inside the source directory, you can change the permissions to any user id with
```

Comment:
And also inside the container? Please give full example of how to fix wrong permissions.

Reply:
I did not need to do anything from within the container, so I am not sure what you mean.